git-cms-merge-topic

```diff
@@ -183,7 +183,7 @@ if [ -z "$NEW_BASE" ]; then
   NEW_BASE=$CMSSW_VERSION
 fi
 if ! [ -d $CMSSW_BASE/src/.git ]; then
-  git cms-init $INITOPTIONS
+  git cms-init --upstream-only $INITOPTIONS
```

Comment:
isn't it going to change the default behaviour of merge-topic and cms-remote commands. Shouldn't `--upstream-only` be added to $INITOPTIONS if passed on the command-line ?

Reply:
Personally, I think it makes sense to setup only `official-cmssw` automatically.
However if others have a strong opinion in this regard, I cam remove this change.

Reply:
I agree that for merge topic it is better to only get the official repo but for git-cms-checkout-topic and  git-cms-rebase-topic (which are alias to git-cms-merge-topic) user most probably need his repo.